executor/executor_bsd.h

```diff
@@ -124,6 +163,7 @@ static void cover_protect(cover_t* cov)
 	if (sysctl(mib, ARRAY_SIZE(mib), &page_size, &len, NULL, 0) != -1)
 		mprotect(cov->data + page_size, mmap_alloc_size - page_size, PROT_READ);
 #endif
+	// TODO(HerrSpace): Implement coverage buffer protection for darwin like in bfb4a51e3
```

Comment:
These probably don't belong unless you already have the code ready.

Reply:
Removed.